chain/config/config_defaults.go

```diff
@@ -10,6 +10,7 @@ func DefaultTestChainConfig() (*TestChainConfig, error) {
 			CheatCodesEnabled: true,
 			EnableFFI:         false,
 		},
+		SkipAccountChecks: true,
```

Comment:
I think by default this should be `false`. I would rather maintain traditional EVM semantics by default.

Reply:
I did it this way bc cheat codes are enabled by default (and I think this makes our prank semantics closer to Foundry). Plus, the error message will probably be perplexing. 

I can change it and maybe catch the error to provide a better message to users

Reply:
That's a fair point - we can enable it by default.